### ISSUE TITLE
Cannot find module 'fs-jetpack' with prisma 3.8.0

### ISSUE THREAD

**Issue by paulrostorp** [prisma/prisma#11112]
### Bug description

In my jest tests, I run `prisma/migrate` programmatically to apply migrations at the beginning of each test suite. Since `v3.8.0` I get the following error:
```
Error: Jest: Got error running globalSetup - [...]/tests/setup.ts, reason: Cannot find module 'fs-jetpack'
Require stack:
- [...]/node_modules/@prisma/sdk/dist/utils/jestContext.js
- [...]/node_modules/@prisma/sdk/dist/index.js
- [...]/node_modules/@prisma/migrate/dist/Migrate.js
- [...]/node_modules/@prisma/migrate/dist/index.js
- [...]/tests/setup.ts
- [...]/node_modules/jest-util/build/requireOrImportModule.js
- [...]/node_modules/jest-util/build/index.js
- [...]/node_modules/jest-config/build/getCacheDirectory.js
- [...]/node_modules/jest-config/build/Defaults.js
- [...]/node_modules/jest-config/build/normalize.js
- [...]/node_modules/jest-config/build/index.js
- [...]/node_modules/jest-cli/build/init/index.js
- [...]/node_modules/jest-cli/build/cli/index.js
- [...]/node_modules/jest-cli/bin/jest.js
- [...]/node_modules/jest/bin/jest.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:941:15)
    at Function.Module._load (node:internal/modules/cjs/loader:774:27)
    at Module.require (node:internal/modules/cjs/loader:1013:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> ([...]/node_modules/@prisma/sdk/dist/utils/jestContext.js:30:36)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Object.newLoader ([...]/node_modules/pirates/lib/index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
```

Weird thing is that `fs-jetpack` doesn't seem to be installed properly... 
```sh
cat node_modules/@prisma/migrate/package.json | grep "fs-jetpack"
    "fs-jetpack": "4.3.0",
```
but then 
```sh
npm ls fs-jetpack
package-name@0.1.0 [...]
└── (empty)
```

### How to reproduce

```js
import { Migrate } from "@prisma/migrate";

/** RUN MIGRATIONS */
const mig = new Migrate();
await mig.applyMigrations();
mig.stop();

```

### Prisma information

all prisma packages @ `v3.8.0`

### Environment & setup

- OS: mac os, mac M1
- Database: PostgreSQL
- Node.js version: v16.1.0

### Prisma Version

```
prisma                  : 3.8.0
@prisma/client          : 3.8.0
Current platform        : darwin-arm64
Query Engine (Node-API) : libquery-engine 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Migration Engine        : migration-engine-cli 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/migration-engine-darwin-arm64)
Introspection Engine    : introspection-core 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/introspection-engine-darwin-arm64)
Format Binary           : prisma-fmt 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules/@prisma/engines/prisma-fmt-darwin-arm64)
Default Engines Hash    : 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f
Studio                  : 0.452.0
Preview Features        : interactiveTransactions
```

**Comment by paulrostorp** [comment]
@janpio FYI the `@prisma/sdk` package also has a dependency on `fs-jetpack` which also fails to add it to my dependency tree

**Comment by Jolg42** [comment]
Hi @paulrostorp 

So the `@prisma/migrate` is not part of the public API, and it looks like we indeed have this problem with `3.8.0`

It's already fixed with https://github.com/prisma/prisma/pull/11111 in our dev package version `3.9.0-dev.6
`, though it's not recommended for production use.

The next official release `3.9.0` will contain the fix.

Note: the feature request about a programmatic API is https://github.com/prisma/prisma/issues/4703

**Comment by paulrostorp** [comment]
ah of course, it was a dev dependency on 3.8. Sounds great, I'll close the issue then

**Comment by Jolg42** [comment]
Note, actually a fix was shipped in 3.8.1 about this.
